src/main/java/org/veupathdb/service/eda/ss/model/variable/binary/AncestorDeserializer.java

```diff
@@ -0,0 +1,38 @@
+package org.veupathdb.service.eda.ss.model.variable.binary;
+
+import org.veupathdb.service.eda.ss.model.variable.VariableValueIdPair;
+
+import java.util.List;
+
+/**
+ * Binary deserializer that knows how to deserialize records with an entity identifier and all of that entity's
+ * ancestors. Rather than returning all ancestors, it only returns a single ancestor specified by the index
+ * in the constructor.
```

Comment:
* Binary deserializer that knows how to deserialize records with an entity idIndex and all of that entity's
* ancestors' idIndexes. It only returns a tuple containing the entity idIndex and the idIndex of the ancestor indicated by [ancestorColumn]

Reply:
Updated